runtime/jvmti/jvmtiExtensionMechanism.c

```diff
@@ -1467,7 +1467,9 @@ jvmtiGetOSThreadID(jvmtiEnv* jvmti_env, ...)
 
 		rc = getVMThread(currentThread, thread, &targetThread, TRUE, TRUE);
 		if (rc == JVMTI_ERROR_NONE) {
-			rv_threadid = (jlong) omrthread_get_osId(targetThread->osThread);
+			if (NULL != targetThread) {
```

Comment:
Shouldn't this check be inside `ifdef` of Java 19 ?

Reply:
Updated.